prime-router/metadata/hl7_mapping/ORU_R01/base/datatype/xad-extended-address.yml

```diff
@@ -30,8 +30,12 @@ elements:
 
   - name: xad-use
     condition: '%resource.use.exists()'
-    value: [ '%resource.use.value.getAddressUse()' ]
+    value: [ '%resource.use.value' ]
     hl7Spec: [ '%{hl7XADField}-7' ]
+    valueSet:
+      home: H
```

Comment:
Add the rest of the values that we know we can convert.  We want to start being as complete as we can to support any kind of data even if we do not have sample data for it.

Reply:
done!